### ISSUE TITLE
Text change on Abstract and keywords page in form

### ISSUE THREAD

**Issue by astridu** [sul-dlss/hungry-hungry-hippo#589]
Please change the text at the top of the page to an H2: "Abstract" as well as "Keywords".
Can you also add a little more white space above 'Keywords' to separate it from the section above it?

**Comment by justinlittman** [comment]
@astridu Can you clarify what you mean by "Please change the text at the top of the page to an H2: "Abstract" as well as "Keywords"?

**Comment by justinlittman** [comment]
@astridu Still unclear. This is what we currently have. Can you annotate with what it should be?

<img width="1200" alt="Image" src="https://github.com/user-attachments/assets/4c399897-7476-45d0-8b82-19fed2f506da" />

**Comment by astridu** [comment]
Oh sorry. This looks fine as is.
